app/src/main/java/org/fossasia/pslab/activity/MultimeterActivity.java

```diff
@@ -134,4 +145,42 @@ public void onClick(View v) {
         });
 
     }
+
+    @SuppressLint("ResourceType")
+    public void howToConnectDialog(String title, String intro, int iconID, String desc) {
+        try {
+            final AlertDialog.Builder builder = new AlertDialog.Builder(this);
+            LayoutInflater inflater = getLayoutInflater();
+            View dialogView = inflater.inflate(R.layout.custom_dialog_box, null);
+            final SharedPreferences settings = this.getSharedPreferences(PREFS_NAME, 0);
+            Boolean skipMessage = settings.getBoolean("skipMessage", false);
```

Comment:
I was going through all PRs related to dialog boxes and it seems like everyone's using too abstract variable name for shared preference boolean. Make this instrument specific. `settings.getBoolean("<skipMultimeterMessage>", false);`
I'll request this change in other PRs as well.